dev-vcs/git-remote-hg/git-remote-hg-1.0.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=(python2_7)
+
+DESCRIPTION="Semi-official Mercurial bridge from Git project"
+HOMEPAGE="https://github.com/mnauw/git-remote-hg"
+SRC_URI="https://github.com/mnauw/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+CDEPEND="dev-vcs/git
+	dev-vcs/mercurial"
+
+DEPEND="${CDEPEND}
+	app-text/asciidoc"
+RDEPEND="${CDEPEND}"
+
+# A few tests fail.
+RESTRICT="test"
```

Comment:
How can they fail if you don't have any test support in the first place?

Reply:
There is test support. `FEATURES=test emerge git-remote-hg` runs it.